actors/builtin/multisig/multisig_actor.go

```diff
@@ -217,11 +216,11 @@ func (a *MultiSigActor) SwapSigner(rt vmr.Runtime, params *SwapSignerParams) *vm
 	var st MultiSigActorState
 	rt.State().Transaction(&st, func() interface{} {
 		if !st.isSigner(params.From) {
-			rt.AbortStateMsg("Party not found")
+			rt.Abort(exitcode.ErrNotFound, "Party not found")
 		}
 
 		if !st.isSigner(params.To) {
-			rt.AbortStateMsg("Party already present")
+			rt.Abort(exitcode.ErrIllegalState, "Party already present")
```

Comment:
ErrForbidden?

Reply:
ErrIllegalArgument, and probably same for the one above.